openstudiocore/src/openstudio_lib/ThermalZonesGridView.cpp

```diff
@@ -244,6 +248,12 @@ struct ModelObjectNameSorter{
 
         addSelectColumn(Heading(QString(SELECTED), false, false, checkbox), "Check to select this row");
       }
+      else if (field == RENDERINGCOLOR){
+        addRenderingColorColumn(Heading(QString(RENDERINGCOLOR), true, false),
```

Comment:
Does this mean that apply to selected does not work on this column?

Reply:
I didn't add an Apply to Selected for a few reasons:

* SpaceType tab didn't have it
* Clicking on the color brings up the Color picker, so it's hard to "selected" a row
* My use case for picking colors is usually to specifically make them look different for rendering purposes, so I didn't think this was necessary
* I hadn't done PR #3256 where I added "Apply to selected" to checkboxes yet, so I thought about it even less

Do you see a use case for having an "Apply to Selected" functionality for color pickers?

Reply:
No, I just wanted to make sure we considered it, plus I had just looked at the other PR